framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -162,6 +162,46 @@ int cfg_get_certification_mode()
     return retVal;
 }
 
+int cfg_get_load_steer_on_vaps(int num_of_interfaces,
+                               char load_steer_on_vaps[BPL_LOAD_STEER_ON_VAPS_LEN])
```

Comment:
Why couldn't load_steer_on_vaps be a `string &` or an std::array?

You would also avoid the need for `load_steer_on_vaps_str`. Or is it intentional and you really want a copy?

Reply:
it is intentional since the BPL provides a c-like interface and thus no `std::string` or `std::array`.
the upper layer that utilizes the BPL saves the information as a string.